frontend/src/scenes/plugins/Plugins.tsx

```diff
@@ -49,14 +50,16 @@ function _Plugins(): JSX.Element {
 
             {user.team?.plugins_opt_in ? (
                 <>
-                    {user.plugin_access.install ? (
+                    {user.organization?.plugins_access_level >= PluginsAccessLevel.Install ? (
                         <Tabs activeKey={pluginTab} onChange={(activeKey) => setPluginTab(activeKey as PluginTab)}>
                             <TabPane tab="Installed" key={PluginTab.Installed}>
                                 <InstalledTab />
                             </TabPane>
-                            <TabPane tab="Repository" key={PluginTab.Repository}>
-                                <RepositoryTab />
-                            </TabPane>
+                            {user.organization?.plugins_access_level === PluginsAccessLevel.Root && (
```

Comment:
Should be install level

Reply:
Rationale in comment.

Reply:
well, but this disables it also for self hosted users :)